src/gsi/get_gefs_for_regional.f90

```diff
@@ -376,16 +376,14 @@ subroutine get_gefs_for_regional
         gfshead%idsl= 1
         gfshead%idvc = 2
 
-        ! FV3GFS write component does not include JCAP, infer from DIMY-2
-        njcap=latb-2
 
         nlat_gfs=gfshead%latb+2
         nlon_gfs=gfshead%lonb
         nsig_gfs=gfshead%levs
 
         jcap_gfs=gfshead%latb-2
 
-        if (mype==mype_out) write(6,*)'GESINFO:  Read NCEP FV3GFS netCDF ', &
```

Comment:
With the removal of mype_out on line 388/386, please remove the variable declaration on line 119.  This variable was noted as being unused while compiling in debug mode.

Reply:
Thanks, Li!